static/js/public/fullRecord.js

```diff
@@ -73,6 +74,24 @@ define('fullRecord', ['module', 'jquery', 'JP2Viewer', 'StructureView', 'dataTab
 		});
 	}
 
+	if ($modsDisplay.length > 0) {
+		$.ajax({
+			url: "/record/" + $modsDisplay.attr("data-pid") + "/metadataView",
+			dataType: "html",
+			success: function(data) {
+				if (/^no.metadata/i.test($.trim(data))) {
```

Comment:
Might make more sense to return a response status of 404, but if it doesn't seem worth while at this time, we can live with it.